src/main/java/com/sampleweb/pages/CarAndMotorbikePage.java

```diff
@@ -0,0 +1,101 @@
+package com.sampleweb.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+// A wild lenny appears ( ͡° ͜ʖ ͡°)
+public class CarAndMotorbikePage extends HomePage {
+
+    public static String PATH = "https://www.amazon.co.uk/"; // Amazon home page url
+    public String title = "Car & Motorbike";
+    public String pageTitle = "//h1[text()=\"Car & Motorbike\"]";
+    public String primeLabel = "//*[@aria-label='Amazon Prime']";
+
+    // TODO: Change file paths from absolute to relative
+    public String navTitle = "//*[@id=\"nav-subnav\"]/a[1]/span";
+    public String dropdownOption = "//*[@id=\"searchDropdownBox\"]/option[11]";
+    public String secondLineMenuLink = "//*[@id=\"nav-subnav\"]/a[1]/span";
+    public String globalStoreTitle = "//*[@id=\"leftNav\"]/h4[2]";
+    public String avgCustomerReviewTitle = "//*[@id=\"leftNav\"]/h4[6]";
+
+    public String resultsList = "//*[@id=\"search\"]/div[1]/div[2]/div/span[3]/div[1]";
+    public String primeCheckbox = "//*[@id=\"leftNav\"]/ul[4]/div/li[1]/span/span/div/label/input";
+
```

Comment:
Maybe declare these variables inside their methods when they are needed. Some of them are not used globally.